docs/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,189 @@
+
+### Creating the Docker images
+Using the `dockerTask.sh` script, we can build or compose the image and container for the **api** application for a specific environment. Build the **debug** image by running the following command.
+
+```bash
+./dockerTask.sh build debug
+```
+
+The image will build the ASP.NET application, run `dotnet restore`, add the debugger to the image, set an `ENTRYPOINT` and finally copy the app to the image. The result is a Docker image named *api* with a `TAG` of *debug*.  See the images on the machine using `docker images`.
+
+```bash
+docker images
+
+REPOSITORY          TAG                  IMAGE ID            CREATED             SIZE
+api                 debug                70e89fbc5dbe        a few seconds ago   779.8 MB
+```
+
+Another way to generate the image and run the application within the Docker container is to open the application in Visual Studio Code and use the debugging tools. 
+
+Select the debugging icon in the View Bar on the left side of VS Code.
+
+![vscode debugging icon](./media/building-net-docker-images/debugging_debugicon.png)
+
+Then tap the play icon or F5 to generate the image and start the application within the container. The Web API will be launched using your default web browser at http://localhost:5000.
+
+![VSCode Docker Tools Debug](./media/building-net-docker-images/docker-tools-vscode-f5.png)
+
+You may set break points in your application, step through, etc. just as if the application was running locally on your development machine as opposed to inside the container. The benefit to debugging within the container is this is the same image that would be deployed to a production environment.
+
+Creating the release image requires simply running the command from the terminal passing the `release` environment name.
+
+```bash
+./dockerTask build release
+```
+
+The command creates the image based on the smaller **microsoft/dotnet:core** base image, `EXPOSE` port 5000, sets the `ENTRYPOINT` for `dotnet api.dll` and copies it to the `/app` directory. There is no debugger, SDK or `dotnet restore` resulting in a much smaller image. The image is named **api** with a `TAG` of **latest**.
+
+```
+REPOSITORY          TAG                  IMAGE ID            CREATED             SIZE
+api                 debug                70e89fbc5dbe        1 hour ago        779.8 MB
+api                 latest               ef17184c8de6        1 hour ago        260.7 MB
+```
+
+## Summary
+
+Using the Docker generator to add the necessary files to our Web API application made the process simple to create the debug and release versions of the images.  The tooling is cross platform by also providing a PowerShell script to accomplish the same results on Windows and Visual Studio Code integration allowed for step through debugging of the application within the container. 
```

Comment:
You should reference the names of the powershell scripts above where you show the bash scripts. I was left wondering if powershell was supported, or if I would have to understand the bash scripts and convert by hand.